Update: Shopify DOC (#6146)
vuestorefront/vue-storefront: packages/core/docs/shopify/README.md

# Shopify integration for Vue Storefront 2

:::tip There's more

> This page only contains the documentation specific to shopify integration. For more information, see the [Vue Storefront documentation](https://docs.vuestorefront.io/v2/).

Reviewer: ```suggestion
This page only contains the documentation specific to Shopify integration. For more information, see the [Vue Storefront documentation](https://docs.vuestorefront.io/v2/).
```

:::

> [Shopify](https://www.vuestorefront.io/shopify) integration for Vue Storefront is open-source and maintained by the superheros from the [Aureate labs](https://aureatelabs.com/).

Reviewer: ```suggestion
[Shopify](https://www.vuestorefront.io/shopify) integration for Vue Storefront is open-source and maintained by the superheroes from the [Aureate labs](https://aureatelabs.com/).
```

> Before diving into it, if you'd like to experience our live [demo](https://shopify-pwa.aureatelabs.com/).

Reviewer: ```suggestion
If you want to preview our product, feel free to visit our [demo](https://shopify-pwa.aureatelabs.com/) application.
```
